Danki2/Assets/Scripts/Actor/Mortal/Components/HealthBar.cs

```diff
@@ -0,0 +1,40 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+public class HealthBar : MonoBehaviour
+{
+    public Image _healthBar;
+    public Mortal _mortal;
+
+    void Update()
+    {
+        if (_healthBar == null || _healthBar.canvas == null)
+        {
+            return;
+        }
+
+        if (_mortal.Health <= 0)
+        {
+            _healthBar.canvas.enabled = false;
+            return;
+        }
+
+        FaceCamera();
+
+        // Update the remaining health display
+        float maxHealth = _mortal.GetStat(Stat.MaxHealth);
+        _healthBar.transform.localScale = new Vector3(_mortal.Health / maxHealth, 1f, 1f);
+    }
+
+    /// <summary>
+    /// Rotate the health bar to face the camera, but lock the y-rotation
+    /// </summary>
+    private void FaceCamera()
```

Comment:
Might want this function to be in it's own script, so we can reuse it for other things. I can imagine a few things wanting this behaviour.